helm/azure-operator/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 name: azure-operator
 version: [[ .Version ]]
 description: "Azure operator to create k8s clusters on Azure"
-appVersion: "[[ .AppVersion ]]"
+appVersion: [[ .AppVersion ]]
```

Comment:
Do we maybe need quotes here, i.e. `"[[ .AppVersion ]]"`?

Reply:
Dunno, added back just in case